web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -142,6 +142,21 @@
 
   const toggleAssetPath = () => (showAssetPath = !showAssetPath);
 
+  const getCameraSearchHref = (asset: AssetResponseDto) => {
+    const cameraSearchUrl = new URL(AppRoute.SEARCH, globalThis.location.href);
+    cameraSearchUrl.searchParams.set(
+      QueryParameter.QUERY,
+      `{"make":"${asset.exifInfo?.make}","model":"${asset.exifInfo?.model}"}`,
```

Comment:
This'll not work if the make or model have double quotes in them. It's probably best to use `JSON.stringify` for the entire value. Same concerns with `getLensModelSearchHref`.

I also think this'll give you unexpected results if the make is unset or if the model is unset, since converting `undefined` to a string gives you the string literal `"undefined"`.

Additionally, you could probably deduplicate the two get*Href functions further since everything but the `value` parameter of the `set` call is the same.

Does a helper function already exist to be able to specify the search URL? Quick digging reveals `getMetadataSearchQuery`, so I imagine you'd want to use that and/or whatever code currently calls this function.